polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaInterpreter.scala

```diff
@@ -150,19 +153,11 @@ class ScalaInterpreter(
                       runtimeTools.eval(tree) -> typ
                     }
                     val methodType = importFromRuntime.importType(fnType)
-                    Some(ResultValue(kernelContext)(method.name.toString, methodType, runtimeFn, cell))
-
-                }.flatten
 
-                val maybeOutput = syms.find(_.name.startsWith("res"))
+                    // I guess we're saying a "def" shouldn't become the cell result?
```

Comment:
I think that makes sense, given that a def will already have a name associated with it